### PR TITLE
New Autodesk CIP suppression keys

### PR DESCRIPTION
**Pull request by wegotoeleven**
There may be a better way of doing this, as all 3 need to be set in order to properly suppress the Autodesk Customer Interaction Program (privacy policy etc) stuff at startup. Let me know your thoughts!

### CODE REVIEW

**Comment by apizz** [comment]
Have a look at `pfm_conditionals`: https://github.com/ProfileCreator/ProfileManifests/wiki/Conditionals-&-Exclusions.  If all three preferences are required, you can create rules for each preference such that when one is added the other two are automatically added as well.

File: Manifests/ManagedPreferencesApplications/com.autodesk.MC3Framework.plist [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+			<string>always</string>
+			<key>pfm_range_list</key>
+			<array>
+				<integer>0</integer>
+				<integer>1</integer>
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>False</string>
+				<string>True</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.autodesk.MC3Framework.plist:124]: Can we make this more descriptive and use something like Enabled / Disabled?  Same for the other prefs as well.

**Comment by apizz** [comment]
@wegotoeleven take a look at my comments whenever you have a chance.

**Comment by wegotoeleven** [comment]
Sorry! I did see this, and I will take a look as soon as I get the chance to :) I've not forgotten about it!

File: Manifests/ManagedPreferencesApplications/com.autodesk.MC3Framework.plist (hunk 1 above)
Comment (wegotoeleven) [reply Manifests/ManagedPreferencesApplications/com.autodesk.MC3Framework.plist]: More descriptive `pfm_range_list_titles` have been added.

**Comment by apizz** [comment]
Referring back to a previous comment: https://github.com/ProfileCreator/ProfileManifests/pull/347#issuecomment-671948535

> Have a look at pfm_conditionals: https://github.com/ProfileCreator/ProfileManifests/wiki/Conditionals-&-Exclusions. If all three preferences are required, you can create rules for each preference such that when one is added the other two are automatically added as well.

**Comment by apizz** [comment]
Otherwise everything looks good to merge.  Lmk if you want to not add pfm_conditionals

**Comment by wegotoeleven** [comment]
Yo. These are outdated now as they're not required for Autodesk.

**Pull request closed without merging**